code/controllers/master_controller.dm

```diff
@@ -95,10 +93,3 @@ datum/controller/game_controller/proc/setup_objects()
 	log_startup_progress("  Initialized [count] pipe networks in [stop_watch(watch)]s.")
 	
 	log_startup_progress("Finished object initializations in [stop_watch(overwatch)]s.")
-
-datum/controller/game_controller/proc/setup_starlight()

```

Comment:
This actually was important if I recall correctly; I think it had something to do with space being made before the starlight config is loaded